resources/schemas.json

```diff
@@ -64,6 +76,42 @@
         },
         "title": "UserIdentityView",
         "description": "Physical person identification data"
+      },
+      "PostalAddressView": {
+        "type": "object",
+        "properties": {
+          "company": {
+            "type": "string",
+            "description": "The company of the Legal Officer"
+          },
+          "line1": {
+            "type": "string",
+            "description": "First address line"
+          },
+          "line2": {
+            "type": "string",
+            "description": "Second address line"
+          },
+          "postalCode": {
+            "type": "string",
+            "description": "Postal code"
+          },
+          "city": {
+            "type": "string",
+            "description": "City"
+          },
+          "country": {
+            "type": "string",
+            "description": "Country"
+          }
+        },
+        "title": "PostalAddressView",
+        "description": "A postal address"
+      },
+      "LanguageView": {
```

Comment:
Didn't we say we would keep that for later?

Reply:
We left the actual internationalization for later, but still we record the spoken languages(s) as any other information.